openpbs.spec

```diff
@@ -83,9 +83,9 @@ Name: %{pbs_name}
 Version: %{pbs_version}
 Release: %{pbs_release}
 Source0: %{pbs_dist}
-Summary: PBS Professional
+Summary: PBS Workload Manager
```

Comment:
Might we want to simply say "Portable Batch System"?

Reply:
We should try not to use "Portable Batch System", and rather use OpenPBS (or PBS).  In this case, since the filename is openpbs.spec, I think it's fine to use OpenPBS.

Reply:
Agreed, since in this case it impacts the spec file that is specific to OpenPBS.